src/ddspdrum/module.py

```diff
@@ -239,6 +239,38 @@ def make_argument(self, control_as_frequency: np.array):
         up_sampled = self.control_to_sample_rate(control_as_frequency)
         return np.cumsum(2 * np.pi * up_sampled / SAMPLE_RATE)
 
+    def oscillator(self, argument):
```

Comment:
I'm not sure but maybe we make VCO into an abstract class and we have a second subclass that is a SineVCO with this oscillator?

I am also feeling like oscillator should be a param in VCO __init__ but not 100% on it

Reply:
ok let's do that then. not 100% on having oscillator type as a param though. how would that work, structurally?